src/Services/QueuedJobService.php

```diff
@@ -232,6 +259,30 @@ public function startJob($jobDescriptor, $startAfter = null)
         }
     }
 
+    /**
+     * Check if maximum number of jobs are currently initialised.
+     *
+     * @codeCoverageIgnore
```

Comment:
```suggestion
```

We don't make exclusions for code coverage in our open source modules, we prefer to be honest about the missing bits 😆